services/navigation/builtin/builtin.go

```diff
@@ -35,16 +39,30 @@ func init() {
 		) (navigation.Service, error) {
 			return NewBuiltIn(ctx, deps, conf, logger)
 		},
+		AttributeMapConverter: func(attributes rdkutils.AttributeMap) (*Config, error) {
+			b, err := json.Marshal(attributes)
+			if err != nil {
+				return nil, err
+			}
+
+			var fooCfg Config
+			if err := json.Unmarshal(b, &fooCfg); err != nil {
+				return nil, err
+			}
+			return &fooCfg, nil
+		},
 	})
 }
 
 // Config describes how to configure the service.
 type Config struct {
-	Store              navigation.StoreConfig `json:"store"`
-	BaseName           string                 `json:"base"`
-	MovementSensorName string                 `json:"movement_sensor"`
-	DegPerSecDefault   float64                `json:"degs_per_sec"`
-	MMPerSecDefault    float64                `json:"mm_per_sec"`
+	Store              navigation.StoreConfig           `json:"store"`
+	BaseName           string                           `json:"base_name"`
+	MovementSensorName string                           `json:"movement_sensor_name"`
+	MotionServiceName  string                           `json:"motion_service_name"`
+	DegPerSec          float64                          `json:"degs_per_sec"`
```

Comment:
Can we add a comment on here clarifying that these are targets, *not* hard limits?